src/constants/platform/copy.ts

```diff
@@ -1,2 +1,2 @@
 export const GEYSER_FEE_DISCLAIMER =
-  'Geyser charges a 2% operational fee from projects using lightning addresses, and no fees from projects that run their own nodes.'
+  'Geyser charges a 4% operational fee from projects using lightning addresses, and no fees from projects that run their own nodes.'
```

Comment:
Let's create a new variable in the constants that holds the percentage value, it'll make things easier for future updates, if any.
LIGHTNING_FEE_PERCENT = 4 , and use it everywhere, where you  made changes.